content/en/design/apps/tutorials/structuring-interviews.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Empathizing with users"
+linkTitle: Structuring user interviews
+weight: 1
+description: >
+  How to structure user interviews
+relatedContent: >
+  design/personas
+---
+
+
+{{% pageinfo %}}Understanding the people you are designing for is a paramount step in the design process, and so one way to achieve that is through conducting user interviews. This tutorial will help design thinkers to understand users’ experiences from their own point of view in terms of: 
+
+- Why they use products in a certain way
+- How they feel about something
+- How they perform various actions{{% /pageinfo %}}
+
+## Background (current situation, problem, and/or opportunity)
+
+Interviews can be a great way to empathize with your users because interviews can give you an in-depth understanding of the users’ values, perceptions, and experiences. They allow you to ask specific questions, while remaining open to exploring your participants’ points of view. They are also often combined with other user research methods, such as usability tests or surveys, so as to gain deeper insights into objective results by asking a user about them and to elicit the user’s subjective opinion on products or interactions.
+
+## Objectives
+
+- To understand pros and cons of semi structured qualitative interviews
+- To understand when to conduct user interviews
+- To understand how to structure user interviews
+
+## Expected outcome
+
+By the end of the tutorial design thinkers will:
+
+- Gain knowledge on user interviews
+- Apply user interview best practices in their field of work
+
+## Brief overview of key concepts
+
+- As the word implies, semi-structured interviews are somewhat structured in that you prepare a set of topics you would like to cover during the interview, but still open enough that you can follow leads in the conversation and change the order of topics.
+
+- This means you have an interview guide with the questions or themes that you want to talk to the user about, but you are free to change the order of questions or to explore different topics that may arise during the interview. 
+
+- User interviews can be very informative and helpful, but only if they are used correctly and for the right things. It’s important to know what you can expect to get out of interviews and what you shouldn’t expect to get out of interviews.
+
+- User interviews can be conducted for the following reasons:
+  - Exploration-before a clear concept has been defined or before a major redesign.
+  - In combination with user tests and formal experiments
+  - In combination with observations
+
+## User Interview requirements
+
+- A good interview requires preparation and careful consideration on the part of the interviewer.
```

Comment:
In the bulleted lists some items have periods at the end and some don't. I would reco to not use them to be consistent with the rest of the docs :)

Reply:
Updated